Getting_Started.md

```diff
@@ -1,47 +1,51 @@
 # Getting Started with Bunsen
 
-This guide is intended to help you get your first Bunsen environment up and running.
+This guide is intended to help get your Bunsen environment up and running.
 
 # Expectations
 
-This project is simply an ansible playbook that has a certain set of requirements that need to be satisfied moving in.
+This project is an ansible playbook with specific requirements that need to be satisfied before it can be used to provision an environment.
 
 ## Initial requirements
 
-Before you can start provisioning your environment, the following items must be satisfied:
+Before you can start provisioning your environment, the following criteria must be satisfied:
 
-1. An environment must consist of at least 3 different systems (physical or virtual).  It is recommended that you have at least 5 machines, but minimally 3 machines can work.  More information on what the different systems will be doing will be provided later.
-    - Of those 3 machines, one must have at least 280GiB of additional storage in a separate block device (i.e. sdb)
-2. Each machine must be able to communicate with the others in the environment both by IP address and by name (i.e. ossbunsen-resource [10.2.0.5] needs to be able to ssh to ossbunsen-farm-f36-1 [10.2.0.6])
+1. At least 3 individual systems, physical and/or virtual.
+    - It is recommended to have at least 5 systems.
+    - The function of the separate systems are described below.
+    - At least one of the 3 systems must have a minimum of 280 GiB of additional storage in a separate block device.
+        - Generally this is /dev/sdb.
+        - If the system is virtualized using 'kvm', it can be /dev/vdb.
```

Comment:
Bunsen file `provisioning/roles/farm/tasks/storage.yml` has the following:
```
      # Use the auxiliary disk, if it exists.
      - name: Setting scratch volume group and test disk names
        set_fact:
          scratch_vg: scratch
          test_disk: "/dev/{{aux_disk}}"

      vars:
        aux_disk: "{%- if ansible_virtualization_type == 'kvm' -%}
                     vdb
                   {%- else -%}
                     sdb
                   {%- endif -%}"
      when: aux_disk in ansible_devices.keys()
```

which is why my previous comment used "must" be /dev/vdb.  Based on the scenarios for setting `test_disk` (unless I missed something) it appears that if it's kvm and not vdb the provisioning will fail in
```
  - name: Verifying storage variables have been set
    assert:
      that:
        - scratch_vg is defined
        - test_disk is defined
```

Our general experience has been that the various virtualization technologies used automatically, in a two disk system, use either sdb or vdb.  Bunsen was written with that in mind.  Maybe it'd be better to make the description be that it must be a two drive system with the second disk being 280GiB, etc. rather than specifically mentioning device names.

Reply:
When I was calling out a particular device (sdb), it was really just to convey that it was another device, not the driver that was providing it.  Similarly, if you're on kvm, you can still use a different driver for the storage and end up with a `sdb` instead of `vdb`, but it's typical to get `vdb` if you're not messing with things.

I think I'd prefer the current spelling where we say it's generally going to be `/dev/sdb`, and if it's kvm that it can be `vdb`.  Perhaps I can rephrase it to mention using the `virtio driver` rather than kvm.

What do you think?

Reply:
Simply having a second storage device is not sufficient for it to be used; for example, I can create a second storage device as NVMe and it's not touched.  The current implementation of bunsen has a number of fallbacks but they're generally predicated on there being enough free space on the root disk; if the root disk doesn't have enough free space provisioning will fail.

There are, as I see it, three choices:
1. extend bunsen to handle more scenarios w/o requiring specific device names
2. proscribe that in environment X the second device must be named Y
3. rely on our experiences that "the right thing" happens and do not mention any device name

The robustness of option 1 will probably always have some holes.
Option 2 is akin to any other environmental qualification; it might not meet some folks desires but they could then extend bunsen.
Option 3, if someone encountered a problem, could be explained away as we've been using default behavior of various virtualization tech; if they are specifying a non-default behavior that's their issue.  Again, they could extend bunsen.

The engineer in me would like to do option 1, but pragmatically I think either option 2 or 3 is the way to go at present.

Reply:
I think we need to hit the pause button on this discussion.

The primary purpose of this PR is to establish a set of instructions to  use; something we don't have posted yet.

So with all that being said, I think we should go with the easiest option and not be too strict about our spelling.  We can always adjust it after the fact.  If we are going to decide only through the three options you provided, then given these constraints, I would opt for option 3 and just omit the names and let people figure it out the hard way until someone with more time can address it more completely.